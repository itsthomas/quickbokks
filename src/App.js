import './App.css';
import Header from './components/Header/Header';
import Calculator from './components/Calculator/Calculator';
import Faq from './components/Faq/Faq';
import styled from 'styled-components';

const SectionsContainer = styled.div`
  display: grid;
  grid-template-columns: repeat(16, 6.25%);
  grid-template-rows: auto;
  margin-top: 40px;
`;

function App() {
  const faqs = [
    {
        question: 'Praesent fermemtum posuere varius?',
        answer: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
    },
    {
        question: 'Integer semper efficitur lacus, nec semper lectus?',
        answer: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
    },
    {
        question: 'Sed erat dolor, pellentesque eu?',
        answer: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
    },
    {
        question: 'Praesent soliicitudin mauris a mattis?',
        answer: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
    }
];

  return (
  <div className="App">
      <Header/>
      <SectionsContainer>
        <Calculator/>
        <Faq faqs={faqs}/>
      </SectionsContainer>
    </div>
    );
}

export default App;
