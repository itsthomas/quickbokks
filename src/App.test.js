import { render } from '@testing-library/react';
import App from './App';

describe('The App component', () => {
  it('should render', () => {
    const { asFragment } = render(<App />);
    const html = asFragment();
    expect(html).toMatchSnapshot();
  });
});
