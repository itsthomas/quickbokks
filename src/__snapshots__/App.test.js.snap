// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The App component should render 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="sc-bdfBwQ hRfulj"
    >
      <div
        class="sc-gsTCUz dlOAAz"
      >
        <img
          alt="logo"
          src="logo.svg"
        />
      </div>
      <ul
        class="sc-dlfnbm iqhHhO"
      >
        <li>
          <a
            href="#item1"
          >
            Item 1
          </a>
        </li>
        <li>
          <a
            href="#item2"
          >
            Item 2
          </a>
        </li>
        <li>
          <a
            href="#item3"
          >
            Item 3
          </a>
        </li>
      </ul>
    </div>
    <div
      class="sc-fFubgz exASbi"
    >
      <div
        class="sc-hKgILt ekhGYm"
      >
        <div
          class="sc-eCssSg gqHjlA"
        >
          <h1
            class="sc-jSgupP eaJicY"
          >
            The Calculator
          </h1>
          <div
            class="sc-gKsewC klroQY"
          >
            <div
              class="sc-iBPRYJ eFXLlj"
            >
              <label
                for="total"
              >
                Total:
              </label>
              <div
                class="sc-fubCfw cqQIyz"
              >
                <div
                  class="sc-pFZIQ fxgwnC"
                  data-testid="total-value"
                >
                  Â£0
                </div>
              </div>
            </div>
            <div
              class="sc-jrAGrp iUbaSR"
            >
              <label
                for="income-from-sales"
              >
                Income from sales:
              </label>
              <input
                data-testid="total-part"
                id="income-from-sales"
                name="income-from-sales"
                type="number"
                value="0"
              />
            </div>
            <div
              class="sc-jrAGrp iUbaSR"
            >
              <label
                for="commissions"
              >
                Commissions:
              </label>
              <input
                data-testid="total-part"
                id="commissions"
                name="commissions"
                type="number"
                value="0"
              />
            </div>
            <div
              class="sc-jrAGrp iUbaSR"
            >
              <label
                for="construction"
              >
                Construction:
              </label>
              <input
                data-testid="total-part"
                id="construction"
                name="construction"
                type="number"
                value="0"
              />
            </div>
            <div
              class="sc-jrAGrp iUbaSR"
            >
              <label
                for="other-income"
              >
                Other income:
              </label>
              <input
                data-testid="total-part"
                id="other-income"
                name="other-income"
                type="number"
                value="0"
              />
            </div>
          </div>
          <div
            class="sc-kEjbxe ecZjXs"
          >
            <label>
              Share my results
            </label>
            <div
              class="sc-iqHYGH cUbXTg"
            >
              <ul>
                <li>
                  <a
                    href="https://www.facebook.com//"
                    title="Facebook"
                  >
                    <i
                      class="fa fa-facebook"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.linkedin.com/company/"
                    title="LinkedIn"
                  >
                    <i
                      class="fa fa-linkedin"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.pinterest.com/"
                    title="Pinterest"
                  >
                    <i
                      class="fa fa-pinterest"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.instagram.com/"
                    title="Instagram"
                  >
                    <i
                      class="fa fa-instagram"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="https://www.youtube.com/"
                    title="YouTube"
                  >
                    <i
                      class="fa fa-youtube"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="https://twitter.com/"
                    title="Twitter"
                  >
                    <i
                      class="fa fa-envelope"
                    />
                  </a>
                </li>
                <li>
                  <a
                    href="https://twitter.com/"
                    title="Twitter"
                  >
                    <i
                      class="fa fa-link"
                    />
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-bqyKva cOBBWE"
      >
        <div
          class="sc-kstrdz kLDyjG"
        >
          <h2
            class="sc-hBEYos eIFLoC"
          >
            Frequently asked questions
          </h2>
          <ul
            class="sc-fodVxV cwzQTu"
          >
            <hr />
            <li
              class="sc-dQppl JzqCl"
            >
              Praesent fermemtum posuere varius?
            </li>
            <hr />
            <li
              class="sc-dQppl JzqCl"
            >
              Integer semper efficitur lacus, nec semper lectus?
            </li>
            <hr />
            <li
              class="sc-dQppl JzqCl"
            >
              Sed erat dolor, pellentesque eu?
            </li>
            <hr />
            <li
              class="sc-dQppl JzqCl"
            >
              Praesent soliicitudin mauris a mattis?
            </li>
            <hr />
          </ul>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
